quiche/src/recovery/hystart.rs

```diff
@@ -143,11 +141,10 @@ impl Hystart {
 
         // Slow Start.
         if self.css_start_time().is_none() {
-            if cwnd >= (LOW_CWND * max_datagram_size) &&
-                self.rtt_sample_count >= N_RTT_SAMPLE
+            if self.rtt_sample_count >= N_RTT_SAMPLE &&
+                self.current_round_min_rtt != Duration::MAX &&
+                self.last_round_min_rtt != Duration::MAX
             {
-                self.rtt_sample_count = 0;
```

Comment:
This part is removed because it was not in the spec and rtt_sample_count will be reset when a new round starts.